src/index.test.ts

```diff
@@ -1,5 +1,6 @@
-import { functionValidator, validator, Context, Event } from './index';
+import {functionValidator, validator, Context, Event, Callback} from './index';
 import * as nock from 'nock';
+import DoneCallback = jest.DoneCallback;
```

Comment:
what is this notation 😯 
also is this even used?

Reply:
🤷‍♂ magic. I'll remove it.

Reply:
+1'd but ya remove this